service/setup.bash

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this script once to setup a machine for running the fuzzbench service.
+
+# Install a supported python version.
+export PYTHON_VERSION=3.7.6
```

Comment:
3.7.8 is latest stable and should work.

Reply:
I'm using the same thing we use in the images. We should update it there first.